plugins/azure-devops/src/components/BuildTable/BuildTable.test.ts

```diff
@@ -239,3 +243,23 @@ describe('getBuildStateComponent', () => {
     });
   });
 });
+
+describe('getBuildDuration', () => {
```

Comment:
You could add a test which checks the result for a valid start and end time duration.

Reply:
So, I actually did but I had nothing but problems. Either the tests would fail or the changes that got the test working would cause the function to return `0s`. The issues I had was around using `fromISO` or `fromJSDate`. It was like what came from the Azure DevOps API I could not replicate in the test. 

How does this sound: let's get this approved and then I'll submit a PR with the tests and we can work together to figure out what I'm doing wrong. I do like having tests and I'm happy to put in the work on this.

Reply:
With the changes to improve date handling I've added the test in that I was having issues with before so this is now tested properly 👍